src/scenes/endscreen.ts

```diff
@@ -18,12 +18,12 @@ export class EndScreen extends Phaser.Scene {
 
     public create(): void {
         this.displayMessage();
+        PlayerManager.removeId();
 
         const button: Phaser.GameObjects.Image = this.add
             .image(ScreenWidth / 2, ScreenHeight / 1.5, "quit")
             .setOrigin(0.5, 0.5);
         interactify(button, 0.5, () => this.onButton());
-        PlayerManager.delete();
```

Comment:
Why will moving this statement to onButton() delay the display correct message problem?

create() is only called at the moment the game ends, so if we move PlayerManger.delete() to onButton(), before anyone clicks on it the correct message should be displayed, right?

And since create() is only called once on transition, if any other client deletes its player information on server, ping doesnt impact the message display.